openshift/templates/strapi/dc.yaml

```diff
@@ -104,6 +104,15 @@ objects:
       test: false
       triggers:
         - type: ConfigChange
+        - imageChangeParams:
```

Comment:
Does this mean a re-deployment will be automatically triggered each time openshift detects a change in the relevant image?

Reply:
Yes, We should be following similar standards/triggers so I made it similar to react dc.yaml

Reply:
sounds good, just wanted to make sure I was reading it right